packages/esm-service-queues-app/src/patient-queue-metrics/metrics-header.component.tsx

```diff
@@ -20,7 +23,45 @@ const MetricsHeader = () => {
   const navigateToQueueScreen = () => {
     navigate({ to: `${spaBasePath}/service-queues/screen` });
   };
+  if (isTablet || isPhone) {
+    return (
+      <div className={styles.metricsContainer}>
+        <span className={styles.metricsTitle}>{metricsTitle}</span>
+        <UserHasAccess privilege="Emr: View Legacy Interface">
```

Comment:
Nit: Probably worth making this configurable. I don't know that this exact privilege works across different implementations. If it does, disregard my suggestion.

Reply:
Agreed, but this isn't a change from the current behavior, so we should ticket separately I think